xcdat/dataset.py

```diff
@@ -41,6 +41,7 @@ def open_dataset(
     path: str,
     data_var: Optional[str] = None,
     add_bounds: bool = True,
+    bounds_axes: Optional[List[str]] = ["X", "Y"],
```

Comment:
What do you think about combining `add_bounds` and `bounds_axes` into a single parameter (e.g., `add_bounds = ["X, "Y"]`, also accepts `None`)?

**I'll need to think about this more** to see if it is confusing or makes the API less intuitive compared to having two separate parameters.

Reply:
I'd be okay with this (assuming it defaults to `["X", "Y"]`).

Reply:
This would be considered a breaking change IMO. I assume most users keep the default `add_bounds=True` argument, and xCDAT is not at v1.0.0 yet so we can kinda get away with it (for now).

I'll take a look at this in PR #434.

Reply:
Hey @pochedls  and @lee1043, I removed `bounds_axes` and refactored `add_bounds` to accept a list axes string, `None`, and `False`.

I kept `False` as an option for API backwards compatibility. I'm certain people have been using `add_bounds=False` to fix metadata issues and then calling `add_bounds()` or `add_missing_bounds()`.

Refer to this commit diff for more info: https://github.com/xCDAT/xcdat/pull/418/commits/c28b2277e6e46abeaa03241a9d1f0a73074201c8

Reply:
I view this as another improvement to the API I originally proposed. Thanks.